include/envoy/tracing/http_tracer.h

```diff
@@ -39,6 +39,26 @@ struct Decision {
   bool traced;
 };
 
+enum class CustomTagType {
+  LITERAL,
+  ENVIRONMENT,
+  REQUEST_HEADER,
+};
+
+struct CustomTag {
+  CustomTag(const CustomTagType type, const std::string& tag, const std::string& value)
```

Comment:
nit: avoid the use of `const` with non-reference parameter types

Reply:
fixed